Source/OnlineSubsystemEpic/Private/OnlineSubsystemEpicTypes.h

```diff
@@ -318,6 +327,22 @@ class FUniqueNetIdEpic
 	}
 };
 
+/**
+ * An enumeration of the different friendship statuses. Modified from eos_friends_types.h
+ */
+UENUM()
+enum class EFriendStatus : uint8
```

Comment:
You don't need to explicitly mark the enum values with integers. This is done by the compiler.